source/val/function.h

```diff
@@ -174,6 +174,9 @@ class Function {
   /// Returns the block predecessors function for the augmented CFG.
   GetBlocksFunction AugmentedCFGPredecessorsFunction() const;
 
+  /// Returns the control flow nesting depth of the given basic block.
+  int GetBlockDepth(BasicBlock* bb);
```

Comment:
You must say this should only be called after dominators have been computed.

It's discretionary whether you should define exactly what you mean by nesting depth.  Either document it here, or in the function.